src/test/java/com/kaltura/client/test/tests/servicesTests/bookmarkTests/ListTests.java

```diff
@@ -21,18 +22,25 @@
 
 public class ListTests extends BaseTest {
 
-    private String assetId = "606283";
-    private int fileId = 935272;
+    private String assetId;
+    private int fileId;
 
-    private String assetId2 = "606282";
-    private int fileId2 = 935270;
+
+    private String assetId2;
+    private int fileId2;
 
     private List <String> assetList = new ArrayList<>();
 
     @BeforeClass
     private void list_tests_before_class() {
-
+        assetId = "606283";
```

Comment:
we have to:
1) set TODO: in case we going to update soon hardcoded values 
OR
2) use ingest VOD util and then get data from resulted media
OR:
3) use Properties.java file for filling there items that can't be added into system by existed API